src/main/java/org/gridsuite/study/server/service/CaseService.java

```diff
@@ -75,10 +75,11 @@ public void deleteCase(UUID caseUuid) {
         }
     }
 
-    public UUID duplicateCase(UUID caseUuid, Boolean withExpiration) {
+    public UUID duplicateCase(UUID caseUuid, Boolean withExpiration, String caseFormat) {
         String path = UriComponentsBuilder.fromPath(DELIMITER + CASE_API_VERSION + "/cases")
                 .queryParam("duplicateFrom", caseUuid)
                 .queryParam("withExpiration", withExpiration)
+                .queryParam("caseFormat", caseFormat)
```

Comment:
Wouldn't that mean that a powsybl case server PR is also needed ?

Reply:
i forget to clean, done now.
about the error in the logs , i have no idea, i don't have this error
can you please see if you still have it after the changes i made ?

Reply:
nice remark, i cleaned the code and updated the tests

Reply:
i can't reproduce the error log you have. does not seem to be related to the changes in this PR
can you please check again with the current version of the code?